package.json

```diff
@@ -70,9 +71,10 @@
     "lint-staged": "^14.0.0",
     "nixt": "^0.5.1",
     "nyc": "^15.0.0",
+    "patch-package": "^8.0.0",
     "pkg": "^5.8.1",
     "rimraf": "^3.0.2",
-    "semantic-release": "^22.0.5",
+    "semantic-release": "^21.0.0",
```

Comment:
Was this downgrade done on purpose?
If so you might want to pin to that version.

Reply:
Using the latest semantic-release (22.0.5), it prints these warnings: 

> npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'semantic-release@22.0.5',
npm WARN EBADENGINE   required: { node: '^18.17 || >=20.6.1' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@semantic-release/commit-analyzer@11.0.0',
npm WARN EBADENGINE   required: { node: '^18.17 || >=20.6.1' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@semantic-release/npm@11.0.0',
npm WARN EBADENGINE   required: { node: '^18.17 || >=20' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@semantic-release/release-notes-generator@12.0.0',
npm WARN EBADENGINE   required: { node: '^18.17 || >=20.6.1' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'env-ci@10.0.0',
npm WARN EBADENGINE   required: { node: '^18.17 || >=20.6.1' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'npm@10.2.0',
npm WARN EBADENGINE   required: { node: '^18.17.0 || >=20.5.0' },
npm WARN EBADENGINE   current: { node: 'v18.15.0', npm: '9.5.0' }
npm WARN EBADENGINE }

Unfortunately pkg only supports node 18.15.0 right now (see https://github.com/vercel/pkg-fetch/blob/main/patches/patches.json), but from what I can tell, everything works fine even with those warnings so I went ahead and reverted this change